e2e/run-fixture-projects.sh

```diff
@@ -0,0 +1,61 @@
+# !/usr/bin/env bash
+
+#
+# NOTE: you can pass the name of a single fixture-projects folder to this script to execute a single test project.
+# E.g.: ./run-fixture-projects.sh vars
+#
+
+# fail if any commands fails
+set -e
+
+# build hardhat-core
+echo "[e2e] Building and packing hardhat-core"
+cd ../packages/hardhat-core
+pnpm install
+pnpm build
+HARDHAT_TGZ_FILE=$(pnpm pack)
+echo "[e2e] Built $HARDHAT_TGZ_FILE"
+cd - >/dev/null
+
+HARDHAT_VERSION=$(echo $HARDHAT_TGZ_FILE | grep -o -E 'hardhat-[0-9]+\.[0-9]+\.[0-9]+' | sed 's/hardhat-//')
```

Comment:
I'm going to remove this because it's not used anywhere and I'm not sure it's useful.